python-checks/src/test/resources/checks/unusedFunctionParameter/unusedFunctionParameterImport.py

```diff
@@ -9,3 +10,9 @@ def method_defined_in_child_class_only(self, a): # Noncompliant
     def method_defined_in_child_class_only_and_not_used(self, a): # Noncompliant
         #                                                     ^
         return compute()
+
+class ChildFromDuplicated(ParentWithDuplicatedParent):
+
+    def do_something(self, a): # FN SONARPY-2347 ChildFromDuplicated has an unresolved type hierarchy, because of the duplicated parent classes
```

Comment:
I think this is a consequence of SONARPY-1829: `DuplicatedParent`, being a class, is not propagated like other types. In `unusedFunctionParameterImported.py`, the hierarchy of `ParentWithDuplicatedParent` should be clearly defined, as the second definition of `DuplicatedParent` always overrides the first one.
I'm not sure we need SONARPY-2347 as a result. Now that it's here, we can probably mark it as blocked by SONARPY-1829 though (or just add the example in SONARPY-1829 and archive SONARPY-2347 as duplicate).

Reply:
Alright! I canceled the SONARPY-2347 and changed the reference in the unit test to SONARPY-1829. Thanks for noticing it!